src/styles/blocks/card.scss

```diff
@@ -0,0 +1,91 @@
+.product-card {
+  padding: 32px 16px 16px;
+  width: $card-width;
+  border: 1px solid #f3f3f3;
+  border-radius: $border-radius;
+  transition: all 0.2s;
+
+  &:hover {
+    transform: scale(1.1);
```

Comment:
The hover effect uses `transform: scale(1.1)`, which might cause layout shifts. Consider using a smaller scale value or a different effect to avoid disrupting the layout.